index.php

```diff
@@ -0,0 +1,4 @@
+<?php 
+    echo "hello world";
+    a = 1;
```

Comment:
pre mali ito nakalimutan mo yung dollar sign
```suggestion
    $a = 1;
```